### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/kubeedge/website/issues/221

### CODE REVIEW

**Comment by Arhell** [comment]
cc @kevin-wangzefeng

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```
Comment (kevin-wangzefeng) [inline LICENSE:3]: This was from https://www.apache.org/licenses/LICENSE-2.0, we should not change here.

File: LICENSE [hunk 2]
```diff
@@ -192,7 +192,7 @@
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at
 
-       http://www.apache.org/licenses/LICENSE-2.0
+       https://www.apache.org/licenses/LICENSE-2.0
```
Comment (kevin-wangzefeng) [inline LICENSE:195]: This was from https://www.apache.org/licenses/LICENSE-2.0, we should not change here.

File: LICENSE (hunk 1 above)
Comment (Arhell) [reply LICENSE]: Done

File: LICENSE (hunk 2 above)
Comment (Arhell) [reply LICENSE]: Done

**Review by kevin-wangzefeng — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**